Sources/CorePayments/Networking/HTTP.swift

```diff
@@ -24,23 +24,23 @@ class HTTP {
         }
         
         if cachingEnabled, let response = urlCache.cachedResponse(for: urlRequest) {
-            let decodedData = try decoder.decode(T.self, from: response.data)
-            return decodedData
+            if let decodedData = try? decoder.decode(T.self, from: response.data) {
+                return decodedData
```

Comment:
We can actually leave this as it was. Any `throw` from the[ decode function here ](https://github.com/paypal/iOS-SDK/blob/c218afc8c8bbe7daefedd363a94c9eb636da2040/Sources/CorePayments/Networking/APIClientDecoder.swift#L12)will get bubbled up and thrown from `performRequest`

Reply:
Sammy, is that what we want? Do we want it to just throw error or do we just force it to get value again if cached value is not decoded for some reason?

Reply:
We don't want to worry about some weird case where decoding is not successful?

Reply:
It looks like in the code Sammy linked above if decoding is not successful we would always throw an error. Unless there is a specific case that you are thinking of that we aren't handling in the decoder?

Reply:
Not sure, I thought maybe the decoding fails for some reason but since we are only caching valid data, I think I can revert.

Reply:
@KunJeongPark Let's revert this change, and then I can ✅

Reply:
Done